web-app/src/index.js

```diff
@@ -35,10 +36,13 @@ const configureStore = () => {
 
 const isAuthenticated = () => {
   if (!authenticated()) {
-    return <Redirect to="/app" />;
+    if (window.location.pathname !== '/landingpage' && window.location.pathname !== '/develop' && window.location.pathname !== '/app' && window.location.pathname !== '/login' && window.location.pathname.substring(0, 25) !== '/login/callback/facebook/' && window.location.pathname !== '/reset-password' && window.location.pathname.substring(0, 16) !== '/reset-password/' && window.location.pathname !== '/sign-up' && window.location.pathname !== '/you-made-it' && window.location.pathname !== '/start') {
```

Comment:
That has nothing to do with React. Will try to look into it this evening/tomorrow.

Can you already place this only using the branch? Will check if everything works first